### ISSUE TITLE
Не работает прокрутка кнопками громкости и касанием

### ISSUE THREAD

**Issue by GoogleCodeExporter** [JustLoy77/ebookdroid#29]
```
В тестовой версии 1.2 не работает прокрутка 
кнопками громкости и касанием.
```

Original issue reported on code.google.com by `mindphaser2k6` on 28 Aug 2011 at 12:59

**Comment by GoogleCodeExporter** [comment]
```
С кнопками уже понятно.
Tap scroll вкючен в настройках ?
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 28 Aug 2011 at 2:32
- Changed state: **Discussion**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in trunk. Will be available since 1.2-rc2+
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 28 Aug 2011 at 2:46
- Changed state: **Fixed**
